torch/utils/data/_utils/serialization.py

```diff
@@ -26,11 +26,11 @@ def serialize_fn(fn):
     Returns a tuple of serialized function and SerializationType indicating the serialization method.
     """
     try:
-        return pickle.dumps(fn), SerializationType("pickle")
+        return fn, SerializationType("pickle")
```

Comment:
I am making this change based on @ejguan 's comment elsewhere

Reply:
Fixed, I am changing it to check if `fn` is a `lambda` rather than trying to pickle `fn` with `pickle` first.